addons/mail/data/mail_data.xml

```diff
@@ -4,7 +4,6 @@
 
         <record id="base.partner_root" model="res.partner">
             <field name="name">OdooBot</field>
```

Comment:
Is it in scope to change this also from `OdooBot`?

Reply:
This already get overwritten in UDES branding to UDES bot. I found in the UI when you click on the UDES bot profile, there were log messages at the bottom of the screen showing how the email changed (as the various different models were installed) "e.g. root@example.com -> odoobot@example.com -> udesbot@example.com", so I removed the email here to stop reference appearing.